piv/key.go

```diff
@@ -319,20 +319,20 @@ func ykAttest(tx *scTx, slot Slot) (*x509.Certificate, error) {
 		instruction: insAttest,
 		param1:      byte(slot.Key),
 	}
-	resp, err := ykTransmit(tx, cmd)
+	resp, err := tx.Transmit(cmd)
```

Comment:
Why was this changed? Mind reverting the diff?

Reply:
With the addition of errors wrapped by `apduErr`, the logic in `ykTransmit` can be removed.  This leaves `ykTransmit` essentially empty except for the call to `tx.Transmit`.  Would it be helpful to see that reverted?

Reply:
Woops yeah I see that. lgtm.